UHResidentInformationAPI.Tests/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{

```

Comment:
Do we need this file here? I think this should be gitignored?

Reply:
This comes from the IDE.

Reply:
I've added the file to gitignore, let me know if it has been implemented correctly